### DISCUSSION TITLE
v3 카카오톡 프로필 메세지 보내는 방법?

### DISCUSSION THREAD

**Discussion by ha1fstack** [storycraft/node-kakao#238] [General]
v3에서 카카오톡 프로필을 보내려면 어떻게 해야 되나요? 전용 템플릿은 없는거같아서 질문드립니다

**Comment by storycraft** [comment]
v3서 전용 템플릿이 없는 타입의 채팅을 보내려면 ChatManager 객체의 sendRaw 메서드를 사용해야 합니다.
카카오톡 프로필 raw attachment 타이핑은 이곳을 참고하세요.
https://github.com/storycraft/node-kakao/blob/51960f48f23d05134b748cf873f09fdbc7362831/src/chat/attachment/contact.ts#L13
